src/inputs/checkbox-radio.scss

```diff
@@ -30,31 +29,42 @@
 
   .iui-checkbox-checkmark,
   .iui-radio-dot {
-    box-sizing: border-box;
     width: $iui-icons-default;
     height: $iui-icons-default;
     margin: 0;
     display: flex;
+    flex-shrink: 0;
     justify-content: center;
     align-items: center;
-
-    @include themed {
-      border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
-      background-color: t(iui-color-background-1);
+    position: relative;
+    isolation: isolate;
+
+    &::after {
+      content: '';
+      position: absolute;
+      inset: 0;
+      z-index: -1;
+      @include themed {
+        border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
+        background-color: t(iui-color-background-1);
+      }
+      @media (prefers-reduced-motion: no-preference) {
+        transition: border-color $iui-speed-fast ease-out;
```

Comment:
color transition doesn't count as "motion" so this media query is not needed

Reply:
Removed `@media (prefers-reduced-motion: no-preference)`.